shared/references/cce-sle15-avail.txt

```diff
@@ -1,27 +1,3 @@
-CCE-91374-9
-CCE-91375-6
-CCE-91376-4
-CCE-91377-2
-CCE-91378-0
-CCE-91379-8
-CCE-91380-6
-CCE-91381-4
-CCE-91382-2
-CCE-91383-0
-CCE-91384-8
-CCE-91385-5
-CCE-91386-3
-CCE-91387-1
-CCE-91388-9
-CCE-91389-7
-CCE-91390-5
-CCE-91391-3
-CCE-91392-1
-CCE-91393-9
-CCE-91394-7
-CCE-91395-4
-CCE-91396-2
-CCE-91397-0
```

Comment:
You are probably removing more CCEs than necessary in this PR. Could you check, please?